src/SSLClient.h

```diff
@@ -75,7 +75,9 @@ class SSLClient : public Client
   bool verify(const char* fingerprint, const char* domain_name);
   void setHandshakeTimeout(unsigned long handshake_timeout);
   void setClient(Client* client);
-  int setTimeout(uint32_t seconds){ return 0; }
+  void setTimeout(uint32_t milliseconds){ 
```

Comment:
Great work here. Only thing - we should not be putting the implementation in the header file. Please will you adjust so that the setTimout signature is in the .h and the implementation goes into the .cpp file